File: app/src/main/java/org/eclipsesoundscapes/ui/base/BaseActivity.java
package org.eclipsesoundscapes.ui.base;

import android.content.Context;
import androidx.appcompat.app.AppCompatActivity;

import org.eclipsesoundscapes.util.LocaleUtils;

public class BaseActivity extends AppCompatActivity {
    @Override
    protected void attachBaseContext(Context newBase) {
        super.attachBaseContext(LocaleUtils.updateLocale(newBase));
    }
}

File: app/src/main/java/org/eclipsesoundscapes/ui/base/BaseActivity.kt
package org.eclipsesoundscapes.ui.base

import android.content.Context
import androidx.appcompat.app.AppCompatActivity
import org.eclipsesoundscapes.util.LocaleUtils

open class BaseActivity : AppCompatActivity() {
    override fun attachBaseContext(newBase: Context) {
        super.attachBaseContext(LocaleUtils.updateLocale(newBase))
    }
}